Chinese-Manual/README.md

```diff
@@ -0,0 +1,129 @@
+# CloudEvents 中文规范
+
+![CloudEvents logo](https://github.com/cncf/artwork/blob/master/projects/cloudevents/horizontal/color/cloudevents-horizontal-color.png)
+
+Declaration： This manual aims to provide a fast and brief introduction of CloudEvents
+in Chinese for people who are new to CloudEvents.
+Most of the content is translated from the original English version.
+It is strongly recommended to read the English version if you find anything lost in translation.
+
```

Comment:
This section describes why we need CloudEvents in Chinese. Suggest adding a section title (both English and Chinese) to this section.